packages/client/hmi-client/package.json

```diff
@@ -37,6 +37,7 @@
 		"@uncharted.software/facets-core": "3.2.6",
 		"@uncharted.software/facets-plugins": "3.2.5",
 		"@vue/runtime-dom": "3.3.4",
+		"@vueup/vue-quill": "latest",
```

Comment:
please use `yarn add --exact @vueup/vue-quill` to have the a precise package number.